examples/webgl_loader_ifc.html

```diff
@@ -25,7 +25,11 @@
 			{
 				"imports": {
 					"three": "../build/three.module.js",
-					"three/addons/": "./jsm/"
+					"three/addons/": "./jsm/",
+					"three/examples/jsm/utils/BufferGeometryUtils": "./jsm/utils/BufferGeometryUtils.js",
```

Comment:
Why you are separately importing BufferGeometryUtils?

Reply:
Because the IFCLoader needs it, and the importmap is needed for it to work without bundling 🤔 Should I change this?

Reply:
If you're specifying the `three/addons` path in the import map you can import from `three/addons/utils/BufferGeometryUtils.js` instead of adding another import map entry.

Reply:
I see. The current bundle of web-ifc-three imports from `three/examples/jsm/utils/`, so to get rid of this entry in the import map, I would need to update all the imports in the library. Is the `three/examples/` route deprecated? If not, maybe I can change the imports in the web-ifc-three library and get rid of this entry of the importmaps in another PR?

Reply:
> Is the three/examples/ route deprecated?

No, it isn't. We just prefer to use the "addons" term since it's more appropriate than "examples".

Reply:
> If not, maybe I can change the imports in the web-ifc-three library and get rid of this entry of the importmaps in another PR?

👍

Reply:
I use the /examples/jsm folder in three-mesh-bvh, as well, so the import maps in the path tracer example list both three/addons/ and three/examples/, too. At the moment I'm not planning to change the bvh project imports:

https://github.com/mrdoob/three.js/blob/4503ef10b81a00f5c6c64fe9a856881ee31fe6a3/examples/webgl_renderer_pathtracer.html#L45-L55